src/Table.luna

```diff
@@ -58,6 +58,20 @@ class Table:
         TableVal wrapper'
     def fromSchemaAndColumns schema columns:
         self.fromWrapper $ createTableWrapper schema.ptr (columns.each .ptr)
+
+    # Constructing `Table` from a list of `Column` values.
+    # 
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,4,5]
+    # > l2 = [11,12,13,14,15]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" Int64Type l2
+    # > table = Table.fromColumns [col1 , col2]
+    #
+    # `columns`: List of columns to construct the `Table`.
```

Comment:
Can `columns` argument be empty? What if there are two columns with the same name in the list?

Reply:
The only tricky edge case that can break things is passing columns with different rows count. We should decide what to do (throw? trim to shortest? pad with nulls?) — and implement and document this.